records_mover/records/delimited/conversions.py

```diff
@@ -39,16 +39,24 @@
 # account for MM/DD time.
 #
 # https://github.com/bluelabsio/records-mover/issues/75
```

Comment:
This file is another example of something which will need to be converted from mappings to full functions.